Rakefile

```diff
@@ -14,3 +13,13 @@ desc "Run rubocop"
 task :rubocop do
   sh "rubocop"
 end
+
+desc "Bundle all gemfiles"
+task :bundle_all do
+  Bundler.with_original_env do
+    system("which matching_bundle 2>&1") || abort("gem install matching_bundle")
+    Dir["gemfiles/*.gemfile"].each do |gemfile|
```

Comment:
Shouldn't this also include the root Gemfile so it doesn't get out of sync?

Reply:
It cannot include that since that is what is executing the task, so it must have already been bundled